Build doesn't work out of the box

build doesn't work out of the box, it relies on artifactory variables being set that are not documented and don't seem necessary for so someone looking to use the libs only and not publish them.

```
$ ./gradlew clean build test install

FAILURE: Build failed with an exception.

* Where:
Build file '.../Android-Lint-Checks/rxlifecycle/build.gradle' line: 45

* What went wrong:
A problem occurred evaluating project ':rxlifecycle'.
> Could not find property 'artifactory_contextUrl' on project ':rxlifecycle'.

* Try:
Run with --stacktrace option to get the stack trace. Run with --info or --debug option to get more log output.

BUILD FAILED

Total time: 5.1 secs

```

--- a/README.md
+++ b/README.md
@@ -9,13 +9,13 @@ Checks
 **RxLifecycle** - Ensures you are binding your Observable subscriptions to the lifecycle of an Activity, Fragment or View
  
 ~~~gradle
-  compile 'io.vokal.lint:rxlifecycle:1.0.0'
+  compile 'io.vokal.lint:rxlifecycle:1.0.3'
 ~~~
 
 **Todo** - Warns you of `//TODO` comments in your code
  
 ~~~gradle
-  compile 'io.vokal.lint:todo:1.0.0'
+  compile 'io.vokal.lint:todo:1.0.3'
 ~~~
 
 Compiling

--- a/build.gradle
+++ b/build.gradle
@@ -63,7 +63,12 @@ subprojects {
     task aarWrapper << { }
 
     project.afterEvaluate {
-        def aarWrap = parent.project("aarWrapper").tasks.find { it.name == 'assemble' }
+        def aarProj = parent.project("aarWrapper")
+        aarProj.android.libraryVariants.forEach {
+            it.mergedFlavor.setApplicationId(it.applicationId + ".${project.name.toLowerCase()}")
+        }
+
+        def aarWrap = aarProj.tasks.find { it.name == 'assemble' }
         aarWrapper.dependsOn(copyLintJar)
         aarWrapper.dependsOn(aarWrap)
         aarWrapper.dependsOn(copyWrappedAAR)

--- a/gradle.properties
+++ b/gradle.properties
@@ -16,3 +16,8 @@
 # This option should only be used with decoupled projects. More details, visit
 # http://www.gradle.org/docs/current/userguide/multi_project_builds.html#sec:decoupled_projects
 # org.gradle.parallel=true
+
+# artifactory config
+artifactory_contextUrl=
+artifactory_user=
+artifactory_password=

--- a/rxlifecycle/build.gradle
+++ b/rxlifecycle/build.gradle
@@ -18,7 +18,7 @@ dependencies {
     testCompile 'com.android.tools:testutils:24.3.1'
 }
 
-version = '1.0.2'
+version = '1.0.3'
 
 jar {
     baseName project.group.replace(".", "-")
@@ -34,7 +34,7 @@ publishing {
     publications {
         aarWrapper(MavenPublication) {
             version version
-            artifact "build/$project.name-$version-release.aar"
+            artifact "build/libs/$project.name-$version-release.aar"
             artifact sourcesJar
             artifact javadocJar
         }

--- a/todo/build.gradle
+++ b/todo/build.gradle
@@ -19,7 +19,7 @@ dependencies {
     testCompile 'com.android.tools:testutils:24.3.1'
 }
 
-version = '1.0.2'
+version = '1.0.3'
 
 jar {
     baseName project.group.replace(".", "-")
@@ -35,7 +35,7 @@ publishing {
     publications {
         aarWrapper(MavenPublication) {
             version version
-            artifact "build/$project.name-$version-release.aar"
+            artifact "build/libs/$project.name-$version-release.aar"
             artifact sourcesJar
             artifact javadocJar
         }

